src/exit/exit.rs

```diff
@@ -0,0 +1,28 @@
+use c_types::c_int;
+use super::_Exit::_Exit;
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn __funcs_on_exit() {}
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn __stdio_exit() {}
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn _fini() {}
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn __libc_exit_fini() {
+    _fini()
```

Comment:
Works for me. Can you include these links in the comment?

http://git.musl-libc.org/cgit/musl/commit/src/exit/exit.c?id=7586360badcae6e73f04eb1b8189ce630281c4b2
http://git.musl-libc.org/cgit/musl/commit/src/exit/exit.c?id=19caa25d0a8e587bb89b79c3f629085548709dd4